Library/Formula/ponyc.rb

```diff
@@ -0,0 +1,15 @@
+class Ponyc < Formula
+  homepage "http://www.ponylang.org"
+  url "http://www.ponylang.org/releases/source/ponyc-0.1.3.tar.bz2"
+  sha256 "bdf6af248362bcb4b9d3869eca5b17b35ef1212786cc93a10efaace516bb6b3e"
+
+  depends_on "llvm" => ["with-rtti"]
+
+  def install
+    system "make", "install", "prefix=#{prefix}"
+  end
+
+  test do
+    system "#{bin}/ponyc", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!